test/metabase/api/moderation_review_test.clj

```diff
@@ -25,16 +30,23 @@
                 (mt/user-http-request :rasta :post 200 "moderation-review" {:text                "Looks good to me"
                                                                             :moderated_item_id   card-id
                                                                             :moderated_item_type "card"}))))
-        (is (= {:text                "Looks good to me"
-                :moderated_item_id   card-id
-                :moderated_item_type "card"
-                :moderator_id        (mt/user->id :rasta)
-                :status              "verified"}
-               (normalized-response
-                (mt/user-http-request :rasta :post 200 "moderation-review" {:text                "Looks good to me"
-                                                                            :moderated_item_id   card-id
-                                                                            :moderated_item_type "card"
-                                                                            :status              "verified"}))))))))
+        (let [raw-response (mt/user-http-request :rasta :post 200 "moderation-review" {:text                "Looks good to me"
+                                                                                       :moderated_item_id   card-id
+                                                                                       :moderated_item_type "card"
+                                                                                       :status              "verified"})]
+          (is (= {:text                "Looks good to me"
```

Comment:
I still haven't quite figured out this aspect of Clojure well, but I notice this entire set of verification is almost identical to what was added in `comment_test.clj` above, although presumably for creation of a review request (as opposed to a comment).  Still, does it make sense to have a common fn for these assertions?

Reply:
I don't have something profound to say. In testing I tend to err towards _not_ being DRY just to keep things nice and explicit, but as more cases are added I'd definitely support extracting some helper fns.